src/shared/sleep-config.c

```diff
@@ -340,9 +340,9 @@ static int resume_configured(void) {
 
         r = find_default_boot_entry(NULL, NULL, &config, &e);
         if (r == -ENOKEY)
-                log_debug_errno(r, "Cannot find the ESP partition mount point, falling back to other checks.");
+                log_debug("Cannot find the ESP partition mount point, falling back to other checks.");
         else if (r < 0)
-                return log_debug_errno(r, "Cannot read boot configuration from ESP, assuming hibernation is not possible.");
+                log_debug("Cannot read boot configuration from ESP, assuming hibernation is not possible.");
```

Comment:
please do not remove error number passed here. When structured logging is used this will end up in the journald if you leave it it in, and is hence useful information. Hence, when there's an error condition always use log_debug_errno() over log_debug()

Reply:
@poettering Thanks for the pointer: didn't realize the use of the error number outside of a return value. 166b83d reverts to `log_debug_errno`